### PR TITLE
created presentational markdown List component

### PR DESCRIPTION
**Pull request by DannySuarez**
looks good?

### CODE REVIEW

File: src/components/markdown/List.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function List({ markdowns }) {
+  const markdownElements = markdowns.map((markdown) => (
```
Comment (mehtaphysical) [inline src/components/markdown/List.js:5]: Would this show the entire markdown is a list? Your design shows tabs with titles.

Comment (DannySuarez) [reply src/components/markdown/List.js:5]: Edited the code to change the tabs to title links, and created a delete button.

File: src/components/markdown/List.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function List({ markdowns, onDelete }) {
+  const markdownElements = markdowns.map((markdown) => (
+    <li key={markdown.id}>
+      <a>{markdown.title}</a>
```
Comment (Zilula) [inline src/components/markdown/List.js]: A should redirect you to another page unless you plan on using react router dom and such...... maybe a button with an onClick to load a different document

**Comment by Zilula** [comment]
LGTM!

**Pull request merged**